src/config/env.ts

```diff
@@ -1,13 +1,22 @@
-import dotenv from "dotenv";
-import path from "path";
-const nodeEnv = process.env.NODE_ENV;
-const filename = nodeEnv === "test" ? ".env.test" : ".env";
+import dotenv from 'dotenv';
+import path from 'path';
+
+export const envVars = {
+  MICROS_ENV: process.env.MICROS_ENV || 'development',
+  MICROS_SERVICE_VERSION: process.env.MICROS_SERVICE_VERSION,
+  NODE_ENV: process.env.NODE_ENV,
+  SENTRY_DSN: process.env.SENTRY_DSN,
+};
```

Comment:
This export const should probably be exported as part of default.  Something like:
```
export default {
  MICROS_ENV: process.env.MICROS_ENV || 'development',
  MICROS_SERVICE_VERSION: process.env.MICROS_SERVICE_VERSION,
  NODE_ENV: process.env.NODE_ENV,
  SENTRY_DSN: process.env.SENTRY_DSN,
  ...env.parsed
}
```

This way, it captures all environment variables that are set in .env as well :)